### PR TITLE
feat(response): add url and timeout information to original request

### PR DESCRIPTION
**Pull request by PhilippSpo**
@emmenko

### CODE REVIEW

**Comment by butenkor** [comment]
:+1:

**Comment by butenkor** [comment]
Rollout please :8ball:

File: src/coffee/services/base.coffee [hunk 1]
```diff
@@ -541,6 +541,15 @@ class BaseService
             headers: response.headers
       else {}
 
+    options = _.omit(@_rest._options, ['access_token'])
+    if options.config
+      options.config = _.omit(options.config, ['client_id', 'client_secret'])
+
+    originalRequest = _.extend(
+      originalRequest,
+      options
```
Comment (emmenko) [inline src/coffee/services/base.coffee]: I would name this `originalOptions: options`, just to make it explicit and to keep the options grouped.

**Comment by emmenko** [comment]
:+1:

**Pull request merged**